readme.md

```diff
@@ -26,6 +26,7 @@ Run `mvn -f mavendeps/pom.xml p2:site && mvn clean verify` in the root of the pr
 ### Install
 
 * Add the generated update site thats located under `path/to/cloned/project/releng/org.codechecker.eclipse.update/target/repository/` to `Help -> Install New Software...` in Eclipse
+  * Alternatively, extract the release downloaded from Releases page, and use that as an update site.
```

Comment:
Wouldn't github just compress the repository ad a zip file? Can that be used as an update site?